source/spoiler_log.cpp

```diff
@@ -73,38 +73,34 @@ const RandomizerHash& GetRandomizerHash() {
   return randomizerHash;
 }
 
-static void SpoilerLog_SaveLocation(std::string_view loc, std::string_view item) {
-  logtxt += loc;
+static inline void SpoilerLog_AddFormatted(std::string_view header, std::string_view value) {
```

Comment:
inline won't do anything here, as the function is static already, so GCC is already going to be aggressive about inlining it where possible.

Reply:
no harm in being explicit about it though afaik